### DISCUSSION TITLE
What can cause NOT redirecting to mobile app when sendTransaction?

### DISCUSSION THREAD

**Discussion by meri-maki** [wevm/wagmi#2754] [Q&A]
I have a website in production which uses wagmi config, transactions on desktop goes through fine
However, on IOS, after sending transactions I'm not redirected automatically to the wallet app. Transactions appear in the wallet app correctly but I have to open the app by myself (which is not the correct flow)

The most interesting thing that I have a repo with minimal reproductive build that i made for a different issue, it has the same logic and dependencies as the website, but there everything works fine and with no issues

Does anyone know what can possibly cause this?

**Comment by classic-k** [comment]
Hello please did you get any solution to the above.
